packages/core/src/amazonqGumby/errors.ts

```diff
@@ -0,0 +1,37 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+import { ToolkitError } from '../shared/errors'
+
+export class JavaHomeNotSetError extends ToolkitError {
```

Comment:
Possible merge blocker: Based on some of @dhasani23 changes, we might need to extend `Error` Instead of `ToolkitError`

Reply:
This should be fine